src/amo/components/AddonHead/index.js

```diff
@@ -19,11 +19,14 @@ import translate from 'core/i18n/translate';
 import { getPreviewImage } from 'core/imageUtils';
 import { getAddonJsonLinkedData } from 'core/utils/addons';
 import type { AppState } from 'amo/store';
+import type { AddonVersionType } from 'core/reducers/versions';
 import type { I18nType } from 'core/types/i18n';
 import type { AddonType } from 'core/types/addons';
 
 type Props = {|
+  _getAddonJsonLinkedData?: typeof getAddonJsonLinkedData,
   addon: AddonType | null,
+  currentVersion: AddonVersionType | null,
```

Comment:
I explained this a bit at https://github.com/mozilla/addons-frontend/pull/6813#discussion_r232242077, although this case is different, because I am  allowing a `null` for `currentVersion` in the props. In this case I guess I was just thinking that, again, this would always be called from a component that has a `currentVersion` prop, so why not just pass it, rather than ask redux to retrieve it yet again. It seems like less processing to do that, but maybe that's not something to be concerned about. Is our philosophy that we should always retrieve data from the redux store rather than accept it via a prop, even if the called will have the same data in a prop of its own?